lib/parse-comment.js

```diff
@@ -127,7 +127,7 @@ const plugin = {
 
 nlp.plugin(plugin);
 
-function parseAddComment(message, action) {
+function getMatcher(message) {
   const whoMatched = nlp(message)
     .match(`${action} [.]`)
```

Comment:
`action` is no longer defined, the tests fail now. Was that an unintentional change or do we need to change that line, too @Berkmann18?

Reply:
Oops, I missed that bit, it wasn't intentional